includes/tripal_hq_admin_dashboard.form.inc

```diff
@@ -144,14 +137,23 @@ function tripal_hq_admin_dashboard_form(array $form, array &$form_state, $status
     $rows[] = $row;
   }
 
-  $output = theme('table', [
-    'header' => $header,
-    'rows' => $rows,
-  ]);
+  $form['table'] = [
+    '#theme' => 'table',
+    '#header' => $header,
+    '#rows' => $rows,
+    '#empty' => t("There are no $status content submissions."),
```

Comment:
This will stop codeclimate from complaining.
```suggestion
    '#empty' => t('There are no !status content submissions.', ['!status' => $status]),
```

Reply:
Thank you!